include/temp_converters.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+#include <map>
+
```

Comment:
I assume this is good

Reply:
Just a guard to prevent header files from being included redundantly